### PR TITLE
.jazzignore <-> .gitignore

### LINKED ISSUE #66: .jazzignore -> .gitignore
At the moment you have to convert your .jazzignore files by hand.
The tool prompts you to do so at the end of the migration.

The first step would be to do this automatically.
And the final version should also track those files during the migration.

My branch `jazzignore` is intended to hold the implementation

### PR DESCRIPTION
**Pull request by ohumbel**
Keep all the `.jazzignore` files in sync with `.gitignore` files at the same location.
Fixes #66

### CODE REVIEW

**Comment by WtfJoke** [comment]
Thanks as always :+1:  I will have a deeper look probably tomorrow.

File: gitFunctions.py [hunk 1]
```diff
@@ -214,6 +222,60 @@ def splitoutputofgitstatusz(lines, filterprefix=None):
...
+        :param jazzignorelines: the input lines
+        :return: the .gitignore lines
+        """
+        recursive = False
+        gitignorelines = []
+        for line in jazzignorelines:
+            if not line.startswith("#"):
+                line = line.strip()
+                if line.startswith("core.ignore"):
+                    gitignorelines.append('\n')
```
Comment (WtfJoke) [inline gitFunctions.py]: replace with os.linesep

File: gitFunctions.py [hunk 2]
```diff
@@ -214,6 +222,60 @@ def splitoutputofgitstatusz(lines, filterprefix=None):
...
+                    if line.startswith("core.ignore.recursive"):
```
Comment (WtfJoke) [inline gitFunctions.py]: reduce lines to 
```recursive = line.startswith("core.ignore.recursive")```

File: gitFunctions.py [hunk 3]
```diff
@@ -214,6 +222,60 @@ def splitoutputofgitstatusz(lines, filterprefix=None):
...
+                        recursive = True
+                    else:
+                        recursive = False
+                for foundpattern in Commiter.findignorepatternregex.findall(line):
+                    gitignoreline = foundpattern + '\n'
```
Comment (WtfJoke) [inline gitFunctions.py]: replace with os.linesep

**Comment by ohumbel** [comment]
@WtfJoke thanks for taking time and care to read the code!
Improvements applied.

**Comment by WtfJoke** [comment]
Thanks. I actually wanted to implement it while merging, however didnt found time in the end. 
I went to a presentation of a Java user group Karlsruhe about Microservices with Clojure, even won a book :smile:

**Pull request merged**